### PR TITLE
Accessibility check

### PR DESCRIPTION
**Pull request by dev89s**
- [ ] Page title - no issues found
- [ ] Image text alternatives “alt text” - added to the missing alt text
https://github.com/dev89s/portfolio/blob/45d710a19c4f208254433274d8f802e23f06997b/index.html#L72
https://github.com/dev89s/portfolio/blob/45d710a19c4f208254433274d8f802e23f06997b/index.html#L205
- [ ] Headings - no issues found (heading levels was correct)
- [ ] Contrast Ratio - Changed the main color theme of the site to a darker shade, buttons and contact form background.
- [ ] Resize text - no issues found
- [ ] Keyboard access - Fixed the tab navigation in the page
https://github.com/dev89s/portfolio/blob/45d710a19c4f208254433274d8f802e23f06997b/index.html#L21
- [ ] Forms, labels, errors - added labels to form input & disabled view
https://github.com/dev89s/portfolio/blob/45d710a19c4f208254433274d8f802e23f06997b/index.html#L228
https://github.com/dev89s/portfolio/blob/45d710a19c4f208254433274d8f802e23f06997b/index.html#L231
https://github.com/dev89s/portfolio/blob/45d710a19c4f208254433274d8f802e23f06997b/index.html#L233
https://github.com/dev89s/portfolio/blob/45d710a19c4f208254433274d8f802e23f06997b/style.css#L548-L550
- [ ] Moving, Flashing, blinking content - no issues found
- [ ] Basic structure check - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,14 +18,14 @@
   <div class="wrapper">
     <header>
       <nav>
-        <a href="#">
+        <a href="#" tabindex="0">
```
Comment (binhussen) [inline index.html:21]: - Great job for updating most accessibility, I think you missed updating the color contrast [here](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-02-28-at-18-28-36-863?failing) is your result, please update based on this [checker](https://www.aditus.io/button-contrast-checker/) :+1: 
Screenshot from 2023-02-28 21-33-56
- [Optional] Kindly your logo is not part of your navigtion so make it outside `nav ` tag for semantic and user accesiblity reasons:+1:

File: index.html [hunk 2]
```diff
@@ -18,14 +18,14 @@
   <div class="wrapper">
```
Comment (binhussen) [inline index.html:18]: - Kindly a page title, <title> in the <head> of the page adequately and briefly describes the content of the page. 
so please update it for better accessibility :+1: 
- here are good page titles:
 **Sasan portfolio**
 **Emma full-stack webdeveloper**

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @dev89s and @ntimanu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @dev89s, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**